app/policies/agent/organisation_policy.rb

```diff
@@ -17,6 +17,10 @@ def create?
     current_agent.territorial_admin_in?(record.territory)
   end
 
+  def close?
+    new?
```

Comment:
nitpick mais je crois que j’aurais trouvé ça plus clair de remettre le check en dur ici 

```suggestion
    current_agent.territorial_admin_in?(record.territory)
```

ça enlève un peu d’indirection et je ne trouve ça pas évident de lire que « pour fermer il faut les même droits que pour accéder au formulaire de création »

Reply:
hum, je pense qu'on peut plutôt extraire une méthode qui représente le fait qu'un agent est admin de territoire, et utiliser le mot-clé `alias` pour écrire les choses de manière succinte et plus facile à lire.